### ISSUE TITLE
Default to blank document

### ISSUE THREAD

**Issue by corobin** [massivemadness/Brackeys-IDE#49]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
currently, the app opens into a non-functional splash screen, in order to start writing anything you must first create a file on the local filesystem (which also opens the file).

suggestion: open the app to a blank document, so that the user can immediately start typing in it. prompt to save it when leaving the document (such as when closing the app)

**Comment by op-amp** [comment]
Yes this could absolutely be cool cos currently whenever I open the editor wanting to randomly type 1 or 2 lines of code I found myself stuck at the tedious process to navigate to a proper folder and create a new file.

Some more suggestions on this topic:
When the user creates a new file from the "File" drop down menu, a blank document could also be used to let the user start type immediately. However, a dialogue box might still pop up to decide the name of the new file in order for the editor to distinguish its language through its file extension type.
